docker-compose.yml

```diff
@@ -1,5 +1,31 @@
 version: '3'
 services:
+  postgres:
+    image: "postgres"
+    container_name: "postgres"
+    environment:
+      POSTGRES_PASSWORD: docker
+      POSTGRES_DB: postgres
+      PG_DATA: /var/lib/postgresql/data/pgdata
+    ports:
+      - "5432:5432"
+    volumes:
+      ./docker/postgres:/var/lib/postgresql/data
+    restart: always
+
+  mongo:
+    image: "mongo"
+    container_name: "mongo"
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: mongoUser
+      MONGO_INITDB_ROOT_PASSWORD: password
+      MONGO_INITDB_DATABASE: mongo
```

Comment:
Not sure if it's a good idea to leave usernames and passwords in plaintext.

Reply:
Of course! Already fixed